Mage.Sets/src/mage/cards/h/HistoriansWisdom.java

```diff
@@ -76,6 +76,7 @@ enum HistoriansWisdomCondition implements Condition {
     @Override
     public boolean apply(Game game, Ability source) {
         game.applyEffects(); // Make sure +2/+1 buff gets applied first
```

Comment:
wtf, it’s a “conditional” class and call for every etb trigger!? Must be removed anyway from current master (no process, no applyeffects here). Keep it bugged, whatever, but do not call such code in such places.

Reply:
Sure. I didn't write this one, just flagging it as an anomaly. But I'll remove it, ignore the test, and open a separate issue after.

Reply:
Confirmed there are no remaining usages of `game.applyEffects()` in non engine code (AuraReplacementEffect has it, but that's only for internal usage for game rules, called within ContinuousEffects only).